NMS-15756: Allow metadata expressions for jmx-config.xml (#6444)
OpenNMS/opennms: docs/modules/reference/pages/performance-data-collection/collectors/jmx.adoc

If a service requires different configuration, use an entry in `$OPENNMS_HOME/etc/jmx-config.xml` to overwrite it.

NOTE: Metadata expressions can also be used in attributes of the `jmx-config.xml` configuration file.

> This lets the user to also reference credentials stored in the secure credentials.

Reviewer: ```suggestion
This allows you to reference credentials stored in the secure credentials vault.
```
